### ISSUE TITLE
Simulate ending a navigation

### ISSUE THREAD

**Issue by i315315** [i315315/amos2022ws02-automotive-test-app#15]
## User story
1. As a user
2. I want to be able to end a running navigation simulation by pressing a button
3. So that I can quit a navigation simulation and simulate a different use case.

It should be possible to end a navigation immediately, delayed by 30 secs or delayed by 1min.

## Acceptance criteria
* In the navigation context there is a possibility to end the simulation of a navigation immediately.
* In the navigation context there is a possibility to end the simulation of a navigation after 30 seconds or after 1 minute.
* The delayed end of the navigation must also be done when the app is running in the background.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
